openfecli/parameters/plan_network_options.py

```diff
@@ -193,6 +226,34 @@ def load_yaml_planner_options(path: Optional[str], context) -> PlanNetworkOption
     - `generate_minimal_redundant_network`
     - `generate_radial_network`
 
+
+Partial Charges
+===============
+
+Methods
+-------
+
+Supported partial charge method choices are:
+    - ``am1bcc``
+    - ``am1bccelf10`` (only possible if `off_toolkit_backend` in settings is set to `openeye`)
+    - ``nagl``
+    - ``espaloma`` (must have `espaloma charge installed`)
+
+Settings
+--------
+
+The following settings can also be set:
+
+    - ``off_toolkit_backend``: this selects the backend of partial charge generation and can be
+      either ``ambertools`` (default), ``openeye`` or ``rdkit``.
+    - ``number_of_conformer``: the number of conformers to use for partial charge generation.
```

Comment:
```suggestion
    - ``number_of_conformers``: The number of conformers to use for partial charge generation.
```

Where are these options defined?